lib/diagnostic.js

```diff
@@ -1,24 +1,16 @@
 'use strict';
 
-// Write your answers inside this file, at the places where it's indicated by the comments.
 
 // 1. Suppose that we wanted to create a browser-based calendar program for
 // keeping track of things. Identify at least two entites such a program might
 // need and describe attributes and methods of each.
-//
-// EXAMPLE
-// Description: Our app manages car sales.
-// Entity: Car
-// Attributes: make, model, year
-// Methods: drive, brake, park
-//
-// Entity: SalesPerson
-// Attributes: name, commissionRate
-// Methods: sell, call
 
-// Answer Starts Here
-//
-// Answer Ends Here
+
+// Description: Web based calendar
+// Entity: Calendar and Web browser
+// Attributes: date, data, time
+// Methods: events and reminders
+
 
```

Comment:
web browser wouldn't necessarily be an entity of calendar, but I can see where you were going with this.